source/faq.txt

```diff
@@ -47,6 +47,28 @@ have to run on the servers that host your MongoDB instances. When
 (OS) that is different than the OS on the source or destination
 clusters.
 
+What hardware specifications should the destination cluster have?
+-----------------------------------------------------------------
+
+For most migrations, the destination cluster should have higher hardware 
+specifications than the source cluster, including the following properties:
+
+- CPU
+- Memory
+- Disk I/O 
+
+These hardware specifications ensure that the destination cluster can handle 
+``mongosync`` writes and that the sync can keep up with the source cluster's 
+workload.
+
+Additionally, the destination cluster must have enough disk storage to 
```

Comment:
[suggestion]

Consider using `includes/fact-oplog-disk-storage.rst` here. I think the only difference would be the loss of "Additionally".

Reply:
Noted!